packages/schematics/angular/migrations/update-8/index.ts

```diff
@@ -33,6 +33,9 @@ const ruleMapping: {[key: string]: string} = {
   'no-template-call-expression': 'template-no-call-expression',
   'templates-no-negated-async': 'template-no-negated-async',
   'trackBy-function': 'template-use-track-by-function',
+  'no-attribute-parameter-decorator': 'no-attribute-decorator',
+  'component-change-detection': 'prefer-on-push-component-change-detection',
```

Comment:
This rule is not released yet so I think you can get rid of this.

Reply:
Nothing is released yet. I'd add a label "blocked" so we don't merge.

Reply:
I mean that the rule `component-change-detection` doesn't need to be updated because it'll probably only be released on the next major of Codelyzer.